src/adapters/bidderFactory.js

```diff
@@ -341,8 +341,9 @@ export function newBidder(spec) {
         if (!Array.isArray(syncs)) {
           syncs = [syncs];
         }
+        const adapter = spec.adapter === undefined ? spec.code : spec.adapter
```

Comment:
Should this condition check for `null` as well as `undefined`?

Reply:
Added a null check. Better to be on the safe side.